.evergreen/run-tests.sh

```diff
@@ -58,8 +58,13 @@ fi
 
 if [ -z "$PYTHON_BINARY" ]; then
     # Use Python 3 from the server toolchain to test on ARM, POWER or zSeries if a
-    # system python3 doesn't exist. This seems to only be an issue on RHEL 7.x.
+    # system python3 doesn't exist or exists but is older than 3.6.
     PYTHON=$(command -v python3 || command -v /opt/mongodbtoolchain/v2/bin/python3) || true
```

Comment:
I think it might be more clear to split this line up into two checks. The first checks if a new enough python 3 version is provided by the OS. If not, see if a new enough python version is in the server toolchain.

Reply:
Done.